scripts/ci/pre_commit/pre_commit_check_2_1_compatibility.py

```diff
@@ -91,6 +92,15 @@ def _check_file(_file: Path):
                 f"as it is not available in Airflow 2.2[/]"
             )
 
+        if GET_MANDATORY_MATCHER.match(line):
+            errors.append(
+                f"[red]In {_file}:{index} there is a forbidden construct "
+                f"(Airflow 2.3+ only):[/]\n\n"
+                f"{lines[index]}\n\n"
+                f"[yellow]You should not use conf.get_mandatory_value "
+                f"as it is not available in Airflow 2.2[/]"
```

Comment:
Isn't the next provider release should set all providers to Airflow 2.2 as min version?
https://github.com/apache/airflow#support-for-providers

Reply:
Yes. But this change is only available in 2.3 :)

Reply:
There will be a separate change about bumping min version to 2.2 after this one.